packages/search-spa/src/app/search/search.component.html

```diff
@@ -0,0 +1,138 @@
+        <span class="pf-c-spinner pf-m-lg" role="progressbar" aria-valuetext="Loading...">
+          <span class="pf-c-spinner__clipper"></span>
+          <span class="pf-c-spinner__lead-ball"></span>
+          <span class="pf-c-spinner__tail-ball"></span>
+        </span>
+      </div>
+      <div class="pf-c-card pf-m-flat result-view">
+        <div class="pf-c-card__body result-border" *ngFor="let result of searchResults?.response?.docs | filterBy: ['content_type']: this.filteredApps |
+               orderBy: sortOrder | slice: 0:sliceLimit">
+          <p class="timestamp">
+            {{ result?.lastModifiedDate | date:'fullDate' || result?.createdDate | date:'fullDate' }}
+          </p>
+          <a href="{{ result?.uri }}" target="_blank">
+            <p class="search-title">{{ result?.title }}</p>
+          </a>
+          <p class="search-abstract">
+            {{ (result?.abstract.length>200)? (result.abstract | slice:0:200)+'...':(result?.abstract) }}</p>
+          <p><span class="pf-c-badge pf-m-read" *ngFor="let tag of result?.tags">{{ tag }}</span></p>
+        </div>
+        <div class="pf-c-card__body result-border" *ngIf="searchResults?.response?.numFound - sliceLimit > 0">
+            <span class="pf-c-spinner pf-m-lg" role="progressbar" aria-valuetext="Loading..." *ngIf="loading">
+              <span class="pf-c-spinner__clipper"></span>
+              <span class="pf-c-spinner__lead-ball"></span>
+              <span class="pf-c-spinner__tail-ball"></span>
+            </span>
+          <p class="search-title cursor-pointer" (click)="loading=true;showMore();" *ngIf="!loading">Show
+            {{ searchResults?.response?.numFound - sliceLimit }}+ more&nbsp;<i class="fas fa-chevron-down"></i>
+          </p>
+        </div>
+      </div>
+    </div>
+    <!-- <div class="pf-l-grid__item pf-m-3-col pf-m-3-row">
+         <p><strong>Trending Applications</strong></p>
+         <div class="pf-c-card pf-m-compact">
+             <div class="pf-c-card__body">
+                 <div class="pf-c-card__body icon-card">
+                     <img src="assets/icons/feedback.svg" width="30px" alt="Avatar image" /> Feedback
+                 </div>
+             </div>
+         </div>
+         &nbsp;
+         <p><strong>Most Searched Keywords</strong></p>
+         <div class="pf-c-card pf-m-compact">
+             <div class="pf-c-card__body">
+                 <p>Outage Management</p>
+             </div>
+             <div class="pf-c-card__body">
+                 <p>Team Dashboard</p>
+             </div>
+             <div class="pf-c-card__body">
+                 <p>Tools and Catalog</p>
+             </div>
+             <div class="pf-c-card__body">
+                 <p>DSAL</p>
+             </div>
+             <div class="pf-c-card__body">
+                 <p>Feedback</p>
+             </div>
+         </div>
+         </div> -->
```

Comment:
If this is no longer needed, or if it will be changed in the future, can you remove these commented lines?